options.go

```diff
@@ -35,11 +35,12 @@ const (
 
 // Tag and Tracer Attribute keys.
 const (
-	ParentSpanGUIDKey = "parent_span_guid" // ParentSpanGUIDKey is the tag key used to record the relationship between child and parent spans.
-	ComponentNameKey  = "lightstep.component_name"
-	GUIDKey           = "lightstep.guid" // <- runtime guid, not span guid
-	HostnameKey       = "lightstep.hostname"
-	CommandLineKey    = "lightstep.command_line"
+	ParentSpanGUIDKey    = "parent_span_guid" // ParentSpanGUIDKey is the tag key used to record the relationship between child and parent spans.
+	ComponentNameKey     = "lightstep.component_name"
+	GUIDKey              = "lightstep.guid" // <- runtime guid, not span guid
+	HostnameKey          = "lightstep.hostname"
+	CommandLineKey       = "lightstep.command_line"
+	AccessTokenHeaderKey = "LightStep-Access-Token"
```

Comment:
I don't think `options.go` is the right place for this constant. This file is about options on the tracer but this header isn't something that we are making an option. I think `collector_client.go` is a better location. Also, maybe we can use something like https://github.com/lightstep/lightstep-tracer-go/blob/master/collector_client_http.go#L19

Also, let's standardize on using standard header values so `Lightstep-Access-Token` with the `s` lowercased.